scripts/griddle.jsx

```diff
@@ -485,13 +485,29 @@ var Griddle = React.createClass({
         var that = this;
             //get the correct page size
             if(this.state.sortColumn !== "" || this.props.initialSort !== ""){
-                var sortProperty = _filter(this.props.columnMetadata, {columnName: this.state.sortColumn});
-                sortProperty = (sortProperty.length > 0 && sortProperty[0].hasOwnProperty("sortProperty") && sortProperty[0]["sortProperty"]) || null;
-
-                data = sortBy(data, function(item){
-                    return sortProperty ? deep.getAt( item, that.state.sortColumn||that.props.initialSort )[sortProperty] :
-                        deep.getAt( item, that.state.sortColumn||that.props.initialSort );
-                });
+                var sortColumn = _filter(this.props.columnMetadata, {columnName: this.state.sortColumn});
+                var sortProperty = sortColumn.length > 0 && sortColumn[0].hasOwnProperty("sortProperty") && sortColumn[0]["sortProperty"] || null;
```

Comment:
it seems that you can use it when your data contains objects, for example:
```
var data = [{
        "id": 0,
        "name": "Mayer Leonard",
        "address": {
          "city": "Kapowsin",
          "state": "Hawaii"
        },
        "country": "United Kingdom",
        "company": "Ovolo",
        "favoriteNumber": 7
      }, ... ]
```
But, looks like that it not needed anymore, after f3a3d34. From griddle-test.js:
```
it('sets sort filter correctly', function(){
    expect(grid.state.sortColumn).toEqual("");
    grid.changeSort("address.state");
    expect(grid.state.sortColumn).toEqual("address.state");
  });
```